### DISCUSSION TITLE
FullDom_Highres and LINKID

### DISCUSSION THREAD

**Discussion by jbensabat** [NCAR/wrf_hydro_nwm_public#706] [General]
Hello 
I am constructing a WRFHYDRO model of a specific basin. 
in the high resolution files, there is one variable LINKID for which i do not have a clear definition
is the local basin associated to the link in the route_link.nc file or is it the pixelwise mapping of the same link ?
thanks
jac

**Comment by rcabell** [comment]
Hi @jbensabat -- this is a Discussion question, not an Issue, so I've moved it to the GitHub Discussions section of the WRF-Hydro repository. Please use this space for future questions, or send them via email to wrfhydro@ucar.edu.

Please only open Issues for bugs (or feature requests) in the code, and use the template provided when doing so. Thanks!

**Comment by kmsampson** [comment]
If you look at Section 5.4.4 or 6.1 of the WRF-Hydro ArcGIS Pre-processing documentation, it states that LINKID is a grid of the channels with values corresponding to the ID variable ('link') in the RouteLink.nc channel parameter file. This grid will only be populated if you configure your domain for reach-based routing. This grid is the mechanism with which water is transferred between the gridded overland routing scheme and the reach-based (Muskingum, Muskingum-Cunge) channel-routing scheme. So this grid represents both a pixel-wise mapping of the channels in your domain as well as a mapping between the local basin and the reach defined in RouteLink.

**Comment by jbensabat** [comment]
Hi
Thanks 
I was not aware of this distinction.
I do not know whether this is matter of discussion or issue but I would suggest to put the in WRFHYDRO documentation explanations for the variables and their structure. I would provide more consistency or alternatively point to the doc where the info is provided 
best
jac
